docs/trace/exception-handling/README.md

```diff
@@ -0,0 +1,67 @@
+# Exception Handling
+
+## User-handled Exception
+
+The term `User-handled Exception` is used to describe exceptions that are
+handled by the application.
+
+While using `Activity` API, the common pattern would be:
+
+```csharp
+using (var activity = MyActivitySource.StartActivity("Foo"))
+{
+    try
+    {
+        Func();
+    }
+    catch (SomeException ex)
+    {
+        activity?.SetStatus(Status.Error);
+        DoSomething();
+    }
+    catch (Exception)
+    {
+        activity?.SetStatus(Status.Error);
+        throw;
+    }
+}
+```
+
+The above approach could become hard to manage if there are deeply nested
+`Activity` objects, or there are activities created in a 3rd party library.
+
+The following configuration will automatically detect exception and set the
+activity status to `Error`:
+
+```csharp
+Sdk.CreateTracerProviderBuilder(options => {
+    options.SetErrorStatusOnException = true;
+});
+```
+
+A complete example can be found [here](./Program.cs).
+
+Note: this feature is platform dependent as it relies on
```

Comment:
From the doc it looks like its support in all cases except in .NET Core 2.1. 
Do you know if it works in windows only?
While we can generally redirect users to official docs, we can list any obvious things right here in this doc . for example, if this only works in Windows, we can mention it here (along with link to official doc)

Reply:
It works on Mac (x64) and Linux (x86 VM) as I've tested locally (I also created issue #1859 to cover ARM64).